code/modules/antagonists/traitor/uplink_handler.dm

```diff
@@ -179,8 +179,8 @@
 		objective.update_progression_reward()
 
 /datum/uplink_handler/proc/abort_objective(datum/traitor_objective/to_abort)
-	if(istype(to_abort, /datum/traitor_objective/final))
-		return
+//	if(istype(to_abort, /datum/traitor_objective/final)) // SKYRAT EDIT REMOVAL
+//		return // SKYRAT EDIT REMOVAL
```

Comment:
By that I mean, I don't see a reason to comment that out. You commented the objectives datum, commenting this as well just seems nonsensical to me.

Reply:
that's fair